archive/body: Keep description consistent with chainHead (#95)
paritytech/json-rpc-interface-spec: src/api/archive_unstable_body.md

**Parameters**:

> - `hash`: String containing an hexadecimal-encoded hash of the header of the block whose body to fetch.

Reviewer: Nit: Ideally I think it would read 

```suggestion
- `hash`: String containing a hexadecimal-encoded hash of the header of the block whose body will be retrieved.
```

But whatever it is, it's good to be consistent!
